beacon_node/beacon_chain/src/kzg_utils.rs

```diff
@@ -43,6 +44,33 @@ pub fn validate_blob<E: EthSpec>(
     kzg.verify_blob_kzg_proof(&kzg_blob, kzg_commitment, kzg_proof)
 }
 
+// TODO
+pub fn verify_cell_proof_batch<E: EthSpec>(
+    _kzg: &Kzg,
+    _cells: Vec<Cell<E>>,
+    _cell_indices: Vec<u64>,
+    _kzg_proofs: Vec<KzgProof>,
+    _kzg_commitments: Vec<KzgCommitment>,
+) -> Result<(), KzgError> {
+    todo!()
```

Comment:
unwrap

Reply:
Oh yeah, this was a `TODO` and wasn't called. I've implemented it in https://github.com/sigp/lighthouse/pull/7117/commits/7f65c506fc963c89199ef3458dedd58f4cdcb0b9